addons/web/controllers/report.py

```diff
@@ -69,6 +70,7 @@ def report_barcode(self, barcode_type, value, **kwargs):
         'Standard93', 'UPCA', 'USPS_4State'
         :param width: Pixel width of the barcode
         :param height: Pixel height of the barcode
+        :param foramt: Accepted values: 'png' (default) or 'svg'.
```

Comment:
foramt => format at least
```suggestion
        :param format: format of the generated image, 'png' (default) or 'svg'
```

Reply:
🙈 done